### PR TITLE
1 commit for 0.19 fixes

### PR DESCRIPTION
**Pull request by puppetpies**
Hi there,

I've just came across a small compilation problem on the new 0.19 release.

Just replaced the as with : just 3 lines.

Regards,

Brian Hood

### CODE REVIEW

**Comment by waterlink** [comment]
I see a failure on Travis. Wasn't it `thing.as(Type)` instead of `thing : Type` ?

File: src/mysql/connection.cr [hunk 1]
```diff
@@ -21,7 +21,7 @@ module MySQL
     end
 
     def set_option(option : LibMySQL::MySQLOption, value)
-      result = LibMySQL.options(@handle, option, value as Void*)
+      result = LibMySQL.options(@handle, option, value : Void*)
```
Comment (DougEverly) [inline src/mysql/connection.cr]: Try `value.as Void*`

File: src/mysql/types.cr [hunk 2]
```diff
@@ -69,7 +69,7 @@ module MySQL
       end
 
       def to_mysql
-        "'#{String.new(value as Slice(UInt8))}'"
+        "'#{String.new(value : Slice(UInt8))}'"
```
Comment (DougEverly) [inline src/mysql/types.cr]: Try `value.as Slice(UInt8)`

File: src/mysql/types.cr [hunk 3]
```diff
@@ -79,7 +79,7 @@ module MySQL
       end
 
       def to_mysql
-        %{'#{(value as Time).to_utc.to_s("%F %T")}'}
+        %{'#{(value : Time).to_utc.to_s("%F %T")}'}
```
Comment (DougEverly) [inline src/mysql/types.cr]: `value.as Time`

Comment (DougEverly) [inline src/mysql/types.cr]: `as` keyword is now a pseudo-method. Try the above changes.

**Review by DougEverly — CHANGES_REQUESTED** [review]
`as` keyword is now a pseudo-method.

**Comment by puppetpies** [comment]
Interesting i will have a read.

Fix in place.

**Comment by waterlink** [comment]
Thanks!

**Comment by waterlink** [comment]
I have published this as a v0.4.4

**Pull request merged**